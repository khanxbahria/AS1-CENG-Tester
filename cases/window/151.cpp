#include "../../your_code/Window.cpp"
#include "../../your_code/Browser.cpp"
#include "../../your_code/LinkedList.h"
int main() {
{
Window* wind = new Window();
wind->getActiveTab();
Tab S = Tab("zz", "xxx","f");
wind->closeTab();
Tab Z = Tab("r", "mm","ddd");
wind->getActiveTab();
Tab G = Tab("eee", "uu","zzz");
wind->getActiveTab();
Tab H = Tab("uuu", "nn","xxx");
wind->moveActiveTabTo(0);
Tab W = Tab("tt", "ggg","eee");
wind->changeActiveTabTo(3);
Tab Q = Tab("vvv", "dd","c");
wind->changeActiveTabTo(1);
wind->getActiveTab();
Tab I = Tab("x", "x","v");
wind->moveActiveTabTo(0);
wind->moveActiveTabTo(3);
wind->newTab(G);
Tab M = Tab("k", "r","jjj");
wind->getActiveTab();
Tab V = Tab("u", "i","b");
wind->newTab(H);
wind->print();
Tab N = Tab("yyy", "ee","nn");
wind->getActiveTab();
Tab O = Tab("k", "u","e");
wind->isEmpty();
wind->changeActiveTabTo(5);
Tab R = Tab("z", "yy","vvv");
wind->isEmpty();
wind->moveActiveTabTo(2);
Tab C = Tab("yy", "ggg","kk");
wind->changeActiveTabTo(5);
Tab K = Tab("jjj", "aaa","s");
wind->newTab(C);
Tab X = Tab("sss", "y","n");
wind->moveActiveTabTo(0);
wind->moveActiveTabTo(0);
wind->changeActiveTabTo(5);
wind->newTab(I);
Tab A = Tab("cc", "e","rrr");
wind->changeActiveTabTo(3);
wind->closeTab();
Tab T = Tab("ii", "vvv","g");
wind->moveActiveTabTo(5);
Tab P = Tab("kk", "bb","ooo");
wind->newTab(A);
wind->isEmpty();
wind->isEmpty();
wind->changeActiveTabTo(3);
wind->moveActiveTabTo(4);
wind->moveActiveTabTo(5);
wind->getActiveTab();
wind->moveActiveTabTo(2);
Tab F = Tab("i", "qq","tt");
wind->isEmpty();
Tab Y = Tab("uuu", "ff","k");
wind->closeTab();
Tab B = Tab("q", "f","rr");
wind->moveActiveTabTo(1);
wind->moveActiveTabTo(3);
wind->closeTab();
Tab J = Tab("g", "vvv","bbb");
wind->moveActiveTabTo(3);
wind->closeTab();
wind->changeActiveTabTo(0);
Tab D = Tab("zz", "ggg","cc");
wind->changeActiveTabTo(0);
wind->moveActiveTabTo(5);
wind->newTab(B);
wind->moveActiveTabTo(2);
wind->isEmpty();
Tab L = Tab("nnn", "b","hhh");
wind->changeActiveTabTo(3);
wind->moveActiveTabTo(4);
wind->getActiveTab();
Tab U = Tab("ff", "b","ooo");
wind->getActiveTab();
wind->print();
wind->closeTab();
wind->closeTab();
wind->getActiveTab();
wind->isEmpty();
wind->closeTab();
wind->closeTab();
wind->getActiveTab();
wind->closeTab();
wind->newTab(D);
wind->print();
wind->getActiveTab();
wind->print();
wind->moveActiveTabTo(2);
wind->print();
wind->getActiveTab();
wind->isEmpty();
wind->closeTab();
wind->getActiveTab();
wind->getActiveTab();
wind->print();
Tab E = Tab("bbb", "m","kk");
wind->newTab(E);
wind->isEmpty();
wind->isEmpty();
wind->isEmpty();
wind->newTab(F);
wind->moveActiveTabTo(0);
wind->isEmpty();
wind->print();
wind->getActiveTab();
wind->closeTab();
wind->isEmpty();
wind->changeActiveTabTo(1);
wind->closeTab();
wind->closeTab();
wind->newTab(J);
wind->isEmpty();
wind->getActiveTab();
wind->changeActiveTabTo(0);
wind->changeActiveTabTo(1);
wind->changeActiveTabTo(5);
wind->moveActiveTabTo(4);
wind->newTab(K);
wind->isEmpty();
wind->newTab(L);
wind->closeTab();
wind->closeTab();
delete wind;
}
	return 0;
}