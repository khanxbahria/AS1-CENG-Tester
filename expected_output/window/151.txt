The active tab is tab 1
uu
eee
zzz

nn
uuu
xxx

The active tab is tab 1
ggg
yy
kk

f
q
rr

The active tab is tab 0
ggg
zz
cc

The active tab is tab 0
ggg
zz
cc

The active tab is tab 0
ggg
zz
cc

Window is empty
The list is empty.
The active tab is tab 0
qq
i
tt

m
bbb
kk

